Documentation/tutorial/docfx_getting_started.md

```diff
@@ -26,7 +26,7 @@ For detailed description about DFM, please refer to [DFM](../spec/docfx_flavored
 -----------------------
 
 *Step1.* DocFX ships as a [chocolatey package](https://chocolatey.org/packages/docfx).
-Install docfx through [Chocolatey](https://chocolatey.org/install) by calling `cinst docfx -y`.
+Install docfx through [Chocolatey](https://chocolatey.org/install) by calling `choco docfx -y`.
```

Comment:
choco **install** docfx -y?

Reply:
I think without the flag is better since it let's users read the terms and agreements if that's something they are interested in doing.